src/nih_wayfinding/app/scripts/routing/turnamenitys-service.js

```diff
@@ -0,0 +1,76 @@
+
+(function () {
+    'use strict';
+
+    /* ngInject */
+    function TurnAmenitys ($http) {
+
+        var overpassUrl = 'http://overpass-api.de/api/interpreter';
+
+        var module = {
+            get: get,
+            attach: attach
+        };
+
+        return module;
+
+        function overpassQuery(x, y) {
+            var meters = 75;
+            var q = '[out:json];' +
+                    'node(around:' + meters +
+                    ',' + y +
+                    ',' + x +
+                    ')' +
+                    '[amenity][name][amenity!="bicycle_rental"];' +
+                    'out;';
+            return q;
+        }
+
+        /**
+         * Look up an amenity from OpenStreetMap to add to the turn by turn
+         * directions. Returns a promise.
```

Comment:
There's a little discussion on this very topic here: https://github.com/jsdoc3/jsdoc/issues/509